tests/v2/validation/snapshot/snapshot.go

```diff
@@ -308,6 +325,13 @@ func snapshotRestoreRKE2K3S(t *testing.T, client *rancher.Client, podTemplate co
 		}
 	}
 
+	return snapshotStruct{initialKubernetesVersion, initialControlPlaneConcurrencyValue, initialWorkerConcurrencyValue, snapshotToRestore}
+}
+
+func restoreRKE2K3S(t *testing.T, client *rancher.Client, v2prov snapshotStruct, etcdRestore *etcdsnapshot.Config, clusterName, clusterID string) {
```

Comment:
nit on the v2prov name. Since we use that for imported packages, I think it could change here.

Reply:
Okay I can change that